cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -242,7 +217,7 @@ If this is the case for your system, you can simply verify the presence of this
 
 This defense technique is specifically discussed in section 4.3 of [Robust Defenses for Cross-Site Request Forgery](https://seclab.stanford.edu/websec/csrf/csrf.pdf). However, bypasses of this defense using Flash were documented as early as 2008 and again as recently as 2015 by Mathias Karlsson to [exploit a CSRF flaw in Vimeo](https://hackerone.com/reports/44146). Riyaz Walikar from [Appsecco Labs](https://appsecco.com/) was able to exploit this even in early 2019 [using older versions of Firefox](https://blog.appsecco.com/exploiting-csrf-on-json-endpoints-with-flash-and-redirects-681d4ad6b31b) and also in some recent versions of Chrome (with a bit of [user involvement](https://github.com/sp1d3r/swf_json_csrf/blob/master/read.html)). Since we cannot the control the versions of browser end user uses nor depend on it for their security, this technique is not recommended as a primary defense in measure.
 
-Besides any possible future bypasses such as Flash, using a static header will make it easy to exploit other state changing operations in the application (similar to the previous explanation on why ease of exploitation is easier in origin/referrer header check than token based mitigation). Including a random token instead of static header value is more or less equal to the token based approach described in the primary defense section. Developers also need to consider that if you are using this approach in an application with both Ajax calls and form tags, this technique would only help Ajax calls in protecting from CSRF and you would still need protect `<form>` tags with approaches described in this document such as tokens. Setting custom headers on form tags is not possible directly. Also, CORS configuration should also be robust to make this solution work effectively (as custom headers for requests coming from other domains trigger a pre-flight CORS check).
+This technique obviously works for Ajax calls and you have to still need protect `<form>` tags with approaches described in this document such as tokens. Setting custom headers on form tags is not possible directly. Also, CORS configuration should also be robust to make this solution work effectively (as custom headers for requests coming from other domains trigger a pre-flight CORS check).
```

Comment:
*Setting custom headers on form tags is not possible directly*

Explains what are the possibilities to address that or add a ref to the section of the document talking about this.

Reply:
I remember adding this statement long back. I think its safe to remove it.

Reply:
Ok so I will delete this